src/Glob.php

```diff
@@ -313,7 +312,7 @@ public static function getBasePath($glob, $flags = 0)
      */
     public static function toRegEx($glob, $flags = 0, $delimiter = '~')
     {
-        if (!Path::isAbsolute($glob) && false === strpos($glob, '://')) {
+        if (!Path::isAbsolute((string) $glob) && false === strpos($glob, '://')) {
```

Comment:
Not a big fan of these string casts, but they are here for BC - previously, an exception would be thrown, while now a `TypeError` raises if the cast is not here.

We can drop them if we move to native type hints.

Reply:
Maybe add this as a comment?

Reply:
dunno, the idea was to make the types more rigid in a future major perhaps

Reply:
Rolling as-is for now, while #28 will expose most of this stuff later 👍